app/src/main/java/com/cornellappdev/transit/ui/screens/HomeScreen.kt

```diff
@@ -229,74 +228,89 @@ fun HomeScreen(
     }
 
     //SheetState for FavoritesBottomSheet
-    val scaffoldState = rememberBottomSheetScaffoldState(
+    val favoritesSheetState = rememberBottomSheetScaffoldState(
         bottomSheetState = SheetState(
             skipPartiallyExpanded = false,
-            initialValue = SheetValue.PartiallyExpanded,
-            confirmValueChange = { true },
+            initialValue = SheetValue.Expanded,
             skipHiddenState = true
         )
     )
 
+    //sheetState for AddFavorites BottomSheet
+    val addSheetState = rememberBottomSheetScaffoldState(
+        bottomSheetState = SheetState(
+            skipPartiallyExpanded = true,
+            initialValue = SheetValue.Hidden
+        )
+    )
+
     var editState by remember {
         mutableStateOf(false)
     }
     var txt by remember {
```

Comment:
This was from an earlier PR, but a more informative variable name other than txt would be helpful. Ideally we can move most of the bottomsheet UI into a separate private function, but that can be a separate PR for the home screen decluttering.